### DISCUSSION TITLE
Can we have something like Categories Inside Collections?

### DISCUSSION THREAD

**Discussion by narendran-kannan** [avitorio/outstatic#60] [Ideas]
Let me explain my request briefly,
Let's say I write blogs on different categories like Ruby on Rails, and NextJs.
So, I wan to List based on the categories.

Proposal:
1. We can have one more column (category) inside each post just like the title or description (or)
2. We can create the posts inside one more directory, for example If I create a post on category nextjs, the md file can be `posts/nextjs/my-post.md

**Comment by avitorio** [comment]
Hi @narendran-kannan, this is a request that has been brought up before. We plan on adding a [custom fields feature](https://twitter.com/outstatic/status/1591077834314854400) in the near future which should solve the need for categories.

**Reply by thecodedrift** [reply]
To the second part, a "List based on the categories", that will be solved with the metadata DB. so you'll be able to query for a list of posts matching specific metadata. [The proposal is here](https://github.com/avitorio/outstatic/discussions/53) and I'd love your thoughts @narendran-kannan!

**Reply by narendran-kannan** [reply]
Looks cool!!! @avitorio @jakobo
